filechooser-portal/FileChooserDialog.vala

```diff
@@ -473,6 +473,45 @@ public class Files.FileChooserDialog : Hdy.Window, Xdp.Request {
         }
     }
 
+    private void accept () {
+        if (action == Gtk.FileChooserAction.SAVE || action == Gtk.FileChooserAction.CREATE_FOLDER) {
+            // If an existing file would be overwritten, ask for permission first
+
+            assert (select_multiple == false);
```

Comment:
Can this be guaranteed as it is set externally?  Would be better to handle this case?

Reply:
Testing shows that the portal code disallows select_multiple in this case.